features/javascript/request.feature

```diff
@@ -0,0 +1,63 @@
+@javascript
+Feature: Request validation
+
+  Background:
+    Given you call:
+    """
+    const gavel = require('gavel');
```

Comment:
Not so much a question about this PR, but for my own understanding. Why is gavel-spec a separate libary instead of part of gavel.js? The features are coupled to the JS implementation so it doesn't seem beneficial to be a separate repository.

Reply:
There has been a notion in the past to have a single language-agnostic spec that would allow implementations in different languages. As history shows, there's only a JavaScript implementation of Gavel, so it indeed makes sense to move this whole package into `gavel.js` to use during its testing pipeline.

The `javascript` directory indeed contains test suites coupled to `gavel.js`, and shouldn't have been a part of `gavel-spec` (this contradicts it being language-agnostic).

I would be glad to re-arrange the relation between `gavel-spec` and `gavel.js` in the future, out of the scope of this pull request.